code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -207,87 +205,69 @@ var/global/list/bottle_colour_choices = list("Blue" = "#0094FF","Dark Blue" = "#
 	name = "Pill bottle (inaprovaline)"
 	desc = "Contains pills used to stabilize patients."
 
 /obj/item/weapon/storage/pill_bottle/inaprovaline/New()
 	..()
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-	new /obj/item/weapon/reagent_containers/pill/inaprovaline( src )
-
-
+	for (var/x = 1 to 7)
+		new /obj/item/weapon/reagent_containers/pill/inaprovaline(src)
+	
+	
 /obj/item/weapon/storage/pill_bottle/dice
 	name = "bag of dice"
 	desc = "Contains all the luck you'll ever need."
 	icon = 'icons/obj/dice.dmi'
 	icon_state = "dicebag"
 
-
 /obj/item/weapon/storage/pill_bottle/dice/New()
 	..()
 	overlays -= colour_overlay
 	colour_overlay = null
-	new /obj/item/weapon/dice/d4( src )
-	new /obj/item/weapon/dice( src )
-	new /obj/item/weapon/dice/d8( src )
-	new /obj/item/weapon/dice/d10( src )
-	new /obj/item/weapon/dice/d00( src )
-	new /obj/item/weapon/dice/d12( src )
-	new /obj/item/weapon/dice/d20( src )
-
+	new /obj/item/weapon/dice/d4(src)
+	new /obj/item/weapon/dice(src)
+	new /obj/item/weapon/dice/d8(src)
+	new /obj/item/weapon/dice/d10(src)
+	new /obj/item/weapon/dice/d00(src)
+	new /obj/item/weapon/dice/d12(src)
+	new /obj/item/weapon/dice/d20(src)
+
+	
 /obj/item/weapon/storage/pill_bottle/hyperzine
 	name = "Pill bottle (hyperzine)"
 	desc = "Contains pills used to keep you active."
 
 /obj/item/weapon/storage/pill_bottle/hyperzine/New()
 	..()
-	new /obj/item/weapon/reagent_containers/pill/hyperzine( src )
-	new /obj/item/weapon/reagent_containers/pill/hyperzine( src )
-	new /obj/item/weapon/reagent_containers/pill/hyperzine( src )
-	new /obj/item/weapon/reagent_containers/pill/hyperzine( src )
-	new /obj/item/weapon/reagent_containers/pill/hyperzine( src )
-	new /obj/item/weapon/reagent_containers/pill/hyperzine( src )
+	for (var/x = 1 to 6)
+		new /obj/item/weapon/reagent_containers/pill/hyperzine(src)
 
+		
 /obj/item/weapon/storage/pill_bottle/creatine
 	name = "Workout Supplements"
 	desc = "Because working out is far too much effort."
 
 /obj/item/weapon/storage/pill_bottle/creatine/New()
-		..()
-		new /obj/item/weapon/reagent_containers/pill/creatine_safe( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
-		new /obj/item/weapon/reagent_containers/pill/creatine_supplement ( src )
+	..()
+	new /obj/item/weapon/reagent_containers/pill/creatine_safe(src)
+	for (var/x = 1 to 5)
```

Comment:
This is just a nitpick, but could you change these to var/i please, i is the for loop var we use universally.